lua/telescope/finders.lua

```diff
@@ -15,6 +15,40 @@ local _callable_obj = function()
 end
 
 
+local IncrementalFinder = _callable_obj()
+
+function IncrementalFinder:new(opts)
+  opts = opts or {}
+  local obj = setmetatable({
+    entry_maker = opts.entry_maker or make_entry.from_string,
+  }, self)
+
+  self.results = {}
+  self.finished = false
+
+  obj._find = coroutine.wrap(function(finder, _, process_result, process_complete)
+    repeat
+      for _, result in ipairs(self.results) do
+        process_result(finder.entry_maker(result))
+      end
+      finder, _, process_result, process_complete = coroutine.yield()
+    until self.completed
```

Comment:
This should probably be `self.finished` which could be the reason that `process_complete()` is not called and a selection is not set. But i could be wrong. I just can't find another `self.completed`

Reply:
Sigh hooray nil... Thanks, nice catch.